app/controllers/untappdAuth.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const bodyParser = require('body-parser');
+
+function untappdAuth(req, res) {
+
+  let CODE = req.query.code
```

Comment:
Same, should be a const I think.
```suggestion
  const CODE = req.query.code
```